installer_scripts/install-yb-voyager

```diff
@@ -850,6 +854,21 @@ create_gather_assessment_metadata_dir() {
 	fi
 }
 
+create_guardrail_scripts_dir() {
+	scripts_parent_dir="/opt/yb-voyager"
+	scripts_dir_name="guardrails-scripts"
+	sudo mkdir -p $scripts_parent_dir/$scripts_dir_name
+
+	output "Installing the guardrails scripts"
+	if [ "${VERSION}" == "latest" ]
+	then
+		pg_permissions_grant_script_url="https://raw.githubusercontent.com/yugabyte/yb-voyager/refs/heads/main/guardrails-scripts/yb-voyager-pg-grant-migration-permissions.sql"
+		sudo wget -nv -O $scripts_parent_dir/$scripts_dir_name/yb-voyager-pg-grant-migration-permissions.sql $pg_permissions_grant_script_url
+	else
+		sudo cp $REPO_ROOT/guardrails-scripts/yb-voyager-pg-grant-migration-permissions.sql $scripts_parent_dir/$scripts_dir_name/
```

Comment:
And also here, can't we copy the whole `guardrails-scripts` folder? This will help later if we plan to add any other script under this not to do any installation changes. And I think in the packaging you are copying the whole folder so the installer-script should also be the same right?